engines/native/config.go

```diff
@@ -28,6 +29,14 @@ var configSchema = schematypes.Object{
 				Pattern: "^[a-zA-Z0-9_.-]+$",
 			},
 		},
+		"createUser": schematypes.Boolean{
+			MetaData: schematypes.MetaData{
+				Title: "Tells if a user should be created to run a command",
+				Description: `When set to true, a new user is created on the fly to run
+				the command. It runs the command from whitin the user's home directory.
+				If false, the command runs without changing userid`,
+			},
+		},
 	},
 	Required: []string{},
```

Comment:
Maybe require that `createUser` is defined, rather that relying on `false` is the default...
Or modify `go-schema-types` to support default values :)